node/install-dependencies-with-gemfury/action.yml

```diff
@@ -0,0 +1,15 @@
+name: Install Dependencies with Gemfury
+description: Installs dependencies with Triax's Gemfury registry
+
+runs:
+  using: composite
+  steps:
+    - name: Authenticate Gemfury
+      run: echo "//npm-proxy.fury.io/triaxtec/:_authToken=$GEMFURY_PULL_TOKEN" > .npmrc
+      env:
+        GEMFURY_PULL_TOKEN: ${{ secrets.GEMFURY_PULL_TOKEN }}
```

Comment:
Can you use secrets cross-action like this? I thought you had to provide them as an input

Reply:
Good catch - you're correct

Reply:
You think it's worth also making the endpoint configurable? At some point we're going to move our private dependencies to GitHub Packages I think.

Reply:
I think that's worth doing.  I considered it before making this PR, but didn't think there was a high chance would change registries any time soon.  It sounds like that was a poor assumption on my part.

I'll add that as an input.

Reply:
Alright, this is now registry agnostic and using inputs.